### PR TITLE
Add Admin2 to backend

### PR DESCRIPTION
**Pull request by batpad**
Addresses Adds import script, models and API for Admin2 data (without geometries)

## Changes

 - Import script for admin2 CSV
 - Models for Admin2
 - Serializer + API for Admin2
 - Tests

## Checklist 
Things that should succeed before merging.

- [ ] Updated/ran unit tests
- [ ] Updated CHANGELOG.md

@szabozoltan69 am having some trouble getting my test to pass, so might need some help with that - but this seems almost ready as long as we get the test working.

### CODE REVIEW

File: api/management/commands/import_admin2s_csv.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            for row in reader:
+                print(row)
+                district_code = row['distr_code']
+                district = District.objects.get(code=district_code)
+                print(district.name)
+                adm2_code = row['GOadm2code']
+                try:
+                    adm2 = Admin2.objects.get(code=adm2_code)
+                except:
+                    print('HELLO')
```
Comment (frozenhelium) [inline api/management/commands/import_admin2s_csv.py:28]: @szabozoltan69 @batpad we should have removed this before the merge

Comment (szabozoltan69) [reply api/management/commands/import_admin2s_csv.py:28]: Too late, deployment is in progress.

Comment (batpad) [reply api/management/commands/import_admin2s_csv.py:28]: Thanks for the catch @frozenhelium - removed in https://github.com/IFRCGo/go-api/pull/1376

**Comment by szabozoltan69** [comment]
Fix is deployed to staging.

**Pull request merged**